File: platform/sdk/python/examples/graph/graph-009_spark_reader_paged_edges.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from xuanji_sdk_graph import GraphClient

client = GraphClient({"endpoint": "graph.local"})
result = client.sparkReaderPagedEdges(25, "0")
if not result["ok"] or len(result["edges"]) != 25:
    sys.exit(1)
print("XJ-OK: graph-009_spark_reader_paged_edges")
sys.exit(0)

File: platform/sdk/nodejs/examples/graph/graph-009_spark_reader_paged_edges.js
const { GraphClient } = require("../../mox-sdk-graph");
const client = new GraphClient({ endpoint: "graph.local" });
const result = client.sparkReaderPagedEdges(25, "0");
if (!result.ok || result.edges.length !== 25) process.exit(1);
console.log("XJ-OK: graph-009_spark_reader_paged_edges");
process.exit(0);
